lib/features/contextPad/ImprovedContextPad.js

```diff
@@ -31,4 +47,35 @@ ImprovedContextPad.prototype._getPosition = function(target) {
   };
 };
 
-ImprovedContextPad.$inject = [ 'contextPad', 'eventBus' ];
\ No newline at end of file
+ImprovedContextPad.prototype._renderTooltips = function({ current }) {
+
+  const entries = domQueryAll('.entry', current.pad.html);
+
+  entries.forEach((entry)=> {
+
+    const position = getTooltipPosition();
+
+    // remove native tooltip
+    const title = entry.getAttribute('title');
+    entry.setAttribute('data-title', title);
+    entry.removeAttribute('title');
```

Comment:
I see we are removing the title and only add it as a private `data-` field. I wonder if this has implications for accessibility? What do you think about this?
```suggestion
    entry.setAttribute('aria-label', title);
    entry.removeAttribute('title');
```

It is easy for us and might help someone :)

Reply:
I think that's fair enough. Updated